ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java

```diff
@@ -458,8 +445,25 @@ String getInternalColumnForExpression(final Expression expression) {
       return internalNameToIndexMap;
     }
 
-    private Map<String, String> getExpressionToInternalColumnNameMap() {
-      return expressionToInternalColumnNameMap;
+    private Expression resolveToInternal(final Expression exp) {
+      if (exp instanceof FunctionCall) {
```

Comment:
Just a question for myself. If I am not mistaken, this is the main change that adds support for functions, right?

Reply:
Yeah.